benchmarks/load_test/load_test.py

```diff
@@ -0,0 +1,109 @@
+import os
+import requests
+import subprocess
+import yaml
+from collections import namedtuple
+
+import numpy as np
+import jinja2
+import click
+from locust import HttpLocust
+
+TEMPLATE_FILE = "task_set.py.jinja2"
+Action = namedtuple("Action", "name path json")
+
+
+def fetch_metadata(project_name, host, port, ambassador, watchman_port):
+
+    if ambassador:
+        data = requests.get(
+            f"{host}:{port}/gordo/v0/{project_name}/", proxies={"http": None}
+        )
+    else:
+        data = requests.get(f"{host}:{watchman_port}", proxies={"http": None})
+
+    json_data = yaml.safe_load(data.content)
+
+    tags = {}
+    for element in json_data["endpoints"]:
+
+        name = element["endpoint-metadata"]["metadata"]["name"]
+        tag_list = element["endpoint-metadata"]["metadata"]["dataset"].get(
+            "tag_list", []
+        )
+        target_tag_list = element["endpoint-metadata"]["metadata"]["dataset"].get(
+            "target_tag_list", []
+        )
+
+        tags[name] = {
+            "tag_list": len(tag_list),
+            "target_tag_list": len(target_tag_list),
+        }
```

Comment:
If you have access to `gordo_components` you could use something like this:
```python
from gordo_components.client.utils import EndpointMetadata

meta = EndpointMetadata(element)
meta.name
meta.target_tag_list
...
```
And if the metadata changes, no one has to remember that this code would also break. Or at least less likely to break?